### ISSUE TITLE
Firewall setup error upon install (windows)

### ISSUE THREAD

**Issue by Dnominated** [syncthing/syncthing#6034]
Verified on Windows 7 & 10 with Synctrayzor/Syncthing 1.2.2

Not sure if this is a SyncThing issue or a SyncTrayzor issue, with the install on a Windows platform.

I'm currently using SyncTrayzor. So if this specifically does not happen with SyncThing, alone, then please close this out. However, I assume that the installer for SyncTrayzor is pulling from the SyncThing installation instructions, And the 'Windows Firewall with Advanced Security' - the firewall interface under Windows 7, shows the rule concerning 'syncthing.exe', not synctrayzor.

I'm running Windows 7, not 10, but another computer I am administering, is running Windows 10, and I can confirm the same firewall setup error occurs.

TLDR: The firewall settings are by default, set to 'Defer to user' for NAT Transversal, and this causes problems where the two computers cannot connect. This setting needs to be set to 'Allow edge transversal', for both the UDP and the TCP settings in the 'syncthing.exe' under the Inbound Firewall rules, for the clients to connect properly. Once manually changed to 'Allow...', the two computers connect just fine, as well as a third non-windows client can see them properly.

**Comment by AudriusButkevicius** [comment]
We don't setup the firewall rules ourselves specifically, I believe the OS just opens a prompt once we try to open a connection.

@canton7 does synctrayzor do something specific for firewalls?
